src/web/routes/application/flow-control/state-machine/operators.js

```diff
@@ -1,7 +1,27 @@
+const { JOURNEYS_KEY, NEXT_ALLOWED_PATH_KEY, STATE_KEY } = require('./keys')
```

Comment:
How about calling this file `session-accessors.js`?

Reply:
Good naming, I have updated.